### DISCUSSION TITLE
Overlapping VLAN's between Tenants

### DISCUSSION THREAD

**Discussion by TheBirdsNest** [nautobot/nautobot#6244] [Ideas]
I was trying to model VLAN's in my network and notices a scenario that isn't accounted for (although may not be common). 
Currently, a VLAN must have a unique ID per VLAN Group, but what I want is a VLAN that can be unique per tenant...

For example, I have a VLAN Group called "Special Purpose" that accommodates non-standard use-cases.
In that group, one location is using VLAN 98 for Purpose X, and another location is using VLAN 98 for purpose Y.. Both locations are separate tenants.

I want all VLAN's to be part of a group, to easily identity and manage them. I think VLAN's should be unique across VLAN ID, VLAN Group and Tenant, not just VID and Group... 

Thoughts?

**Comment by itdependsnetworks** [comment]
Not my call, but my guess would be this would be something done via Custom vValidators. I think it is fair to say that not having uniqueness across a tenant is valid for many organizations.
